src/tooltips.js

```diff
@@ -78,14 +78,57 @@ module.exports = {
     display: function (gene, div, mouseover) {
       const obj = {};
       obj.header = gene.gene.protid;
+      
+      function fetch_annots(gene_tooltip_obj, protid, mouseover){
+          $.ajax({
+            type:'get',
+            url: 'https://omabrowser.org/api/protein/' + protid + '/gene_ontology/?format=json',
```

Comment:
please use only `/api/protein/...` as the url. this way it will fetch the annotations from the correct instance if deployed on a different instance

Reply:
Done